src/Microsoft.ComponentDetection.Detectors/maven/MavenStyleDependencyGraphParser.cs

```diff
@@ -104,23 +110,40 @@ private void RecordDependencies(int position, string versionedComponent, ISingle
                 tupleStack.Pop();
             }
 
-            var componentAndDevDependencyTuple = MavenParsingUtilities.GenerateDetectedComponentFromMavenString(versionedComponent);
+            var componentAndDevDependencyTuple = MavenParsingUtilities.GenerateDetectedComponentAndMetadataFromMavenString(versionedComponent);
             var newTuple = (ParseLevel: position, componentAndDevDependencyTuple.Component);
-
+            RecordTelemetryIfProvidedScope(componentAndDevDependencyTuple.dependencyScope);
             if (tupleStack.Count > 0)
             {
                 var parent = tupleStack.Peek().Component;
                 componentRecorder.RegisterUsage(parent);
-                componentRecorder.RegisterUsage(newTuple.Component, parentComponentId: parent.Component.Id, isDevelopmentDependency: componentAndDevDependencyTuple.IsDevelopmentDependency);
+                componentRecorder.RegisterUsage(
+                    newTuple.Component,
+                    parentComponentId: parent.Component.Id,
+                    isDevelopmentDependency: componentAndDevDependencyTuple.IsDevelopmentDependency,
+                    dependencyScope: componentAndDevDependencyTuple.dependencyScope);
             }
             else
             {
-                componentRecorder.RegisterUsage(newTuple.Component, isExplicitReferencedDependency: true, parentComponentId: topLevelComponent.Component.Id, isDevelopmentDependency: componentAndDevDependencyTuple.IsDevelopmentDependency);
+                componentRecorder.RegisterUsage(
+                    newTuple.Component,
+                    isExplicitReferencedDependency: true,
+                    parentComponentId: topLevelComponent.Component.Id,
+                    isDevelopmentDependency: componentAndDevDependencyTuple.IsDevelopmentDependency,
+                    dependencyScope: componentAndDevDependencyTuple.dependencyScope);
             }
 
             tupleStack.Push(newTuple);
         }
 
+        private void RecordTelemetryIfProvidedScope(DependencyScope? dependencyScope)
+        {
+            if (dependencyScope.HasValue && (dependencyScope == DependencyScope.Provided || dependencyScope == DependencyScope.System))
+            {
+                using var record = new MavenDetectorProvidedScopeRecord();
```

Comment:
Rather than having a new record for each entry (this might cause hundreds of logs on large repos) I would recommend having a counter in a shared record in `Parse` that is incremented when a provided dependency is found. DockerService and LinuxScanner have examples you can reference